### PR TITLE
Update multi_base.md

### PR DESCRIPTION
**Pull request by ronaldosc**
- update to link in 'multi-namespace' not found reference, and added another from Kubernetes Docs.

### CODE REVIEW

File: site/content/en/guides/example/multi_base.md [hunk 1]
```diff
@@ -173,4 +173,6 @@ spec:
 ```
 
 Similarly to adding different `namePrefix` in different variants, one can also add different `namespace` and compose those variants in
-one _kustomization_. For more details, take a look at [multi-namespaces](multi-namespace.md).
+one _kustomization_. For more details, take a look at:
+- [Setting Namespaces](../config_management/namespaces_names.md)
```
Comment (KnVerey) [inline site/content/en/guides/example/multi_base.md:177]: This link doesn't ascend far enough and is broken. Please check the deploy preview created by the CI checks.

Comment (ronaldosc) [reply site/content/en/guides/example/multi_base.md:177]: Had you checked it in preview ? It's working, I think you saw old CI checks.

Before I was looking for the multi-namespaces file, let me know if you have the link.

Comment (ronaldosc) [reply site/content/en/guides/example/multi_base.md:177]: The link original was broken but this one is similar for explanation

**Comment by ronaldosc** [comment]
remove-lifecycle rotten

**Pull request closed without merging**